src/impress.js

```diff
@@ -6,13 +6,13 @@
  * in modern browsers and inspired by the idea behind prezi.com.
  *
  *
- * Copyright 2011-2012 Bartek Szopka (@bartaz), 2016-2020 Henrik Ingo (@henrikingo)
+ * Copyright 2011-2012 Bartek Szopka (@bartaz), 2016-2023 Henrik Ingo (@henrikingo)
  *
  * Released under the MIT License.
  *
  * ------------------------------------------------
  *  author:  Bartek Szopka, Henrik Ingo
- *  version: 1.1.0
+ *  version: 2.1.0
```

Comment:
Ah yes... You're right that this is how most projects would do it. To be honest here I have kept the number that is the latest release, and then increase at release time, if I remember...

So with that logic this should be 

    2.0.0

I don't feel strongly about this and if anything I'd want this to be scripted instead of manually updated.